test/decl/protocol/invalid_accessors_implementation.swift

```diff
@@ -0,0 +1,38 @@
+// RUN: %target-typecheck-verify-swift
+
+
+//===---
+// SR-13963
+//===---
+
+protocol Protocol1 {
+  var a: Int { get { 0 } } // expected-error {{protocol property cannot have a default implementation of getter}}
```

Comment:
I wonder if it wouldn't make sense(or be possible) to offer a fix-it to remove from lbrace to rbrace  e.g. `{ 0 }` in this case? This is more a question as I'm just thinking it would be a good thing for the user also...

Reply:
I also think it makes sense to offer a fix-it, but I'm not sure how I can get the SourceRange from the lbrace to rbrace without disrupting the ongoing parsing.. I'd really appreciate any feedbacks or pointers on how I could do that! :)

Reply:
I prefer not to offer a fix-it for removing. We should be very cautious about deleting user's code.